com.microsoft.mrtk.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -183,26 +183,34 @@ internal protected XRInteractionManager InteractionManager
         /// <summary>
         /// Registers an interactor to be managed by the interaction mode manager
         /// </summary>
-        /// <param name="controllerInteractor">An XRBaseControllerInteractor which needs to be managed based on interaction modes</param>
-        public void RegisterInteractor(XRBaseControllerInteractor controllerInteractor)
+        /// <param name="interactor">An XRBaseInteractor which needs to be managed based on interaction modes</param>
+        public void RegisterInteractor(XRBaseInteractor interactor)
```

Comment:
Same comment about `XRBaseInteractor` vs `IXRInteractor` (should prefer `IXRInteractor` at all costs)

Reply:
See other comment